### PR TITLE
Clarify keywords

### PR DESCRIPTION
**Pull request by Tratcher**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/extensions/pull/4305)

### CODE REVIEW

**Review by danmoseley — APPROVED** [review]
thanks

File: docs/building.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Here are few commands that you will likely use the most:
     - `build.sh --restore`: to install the required .NET SDK, .NET tools and the toolset. This is equivalent to running `./restore.sh`.
     - `build.sh --build`: to build the solution<sup>1</sup>.
     - `build.sh --test`: to run all unit tests in the solution<sup>1</sup>.
-    - `build.sh --vs <keywords>`: to generate a "filtered" solution and save it as `SDK.sln`. It also performs the "restore" operation. For example: `./build.sh --vs Http,Fakes`.<br />
+    - `build.sh --vs <keywords>`: to generate a "filtered" solution and save it as `SDK.sln`. It also performs the "restore" operation. Keywords can be any part of the name or path of project files you want to include. For example: `./build.sh --vs Http,Fakes,AspNetCore`.<br />
```
Comment (danmoseley) [inline docs/building.md:48]: is it intended that it's --vs here and -vs below

Comment (joperezr) [reply docs/building.md:48]: Good catch, I don't think this is intentional.

Comment (RussKie) [reply docs/building.md:48]: Our bash shell scripts accept a single dash (e.g., `-vs`) for the compat with the ps1 scripts.
https://github.com/dotnet/extensions/blob/c8e34c57199a2e5e3864649f3bd16258b88ff67e/eng/build.sh#L34-L41

**Pull request merged**